packages/gasket-core/lib/index.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-console, no-process-env */
+// @ts-check
```

Comment:
I don't think this is necessary, as the type checking should be enabled already.